### ISSUE TITLE
Feature request for healing after battle

### ISSUE THREAD

**Issue by telostia** [EasyFarm/EasyFarm#242]
Hi, I love your app! thanks for making such an awesome app! :)
There is a feature I think would be beneficial to anyone. The ability to heal after the battle has finnished. Such as a boss to farm af3 seals/jewels etc. but not depended on rest during battle like the one included. resting during battle could get you killed! thanks again.

**Comment by Mykezero** [comment]
Isn't the healing tab responsible for casting healing spells during battle? I think I might not understand your suggestion ^^;

**Comment by Mercurian** [comment]
He wants to make a check box that says like: 

While idle cure to full. He wants to heal, but after the battle ends. The >End option I've never had good results with using.

**Comment by Mykezero** [comment]
That sounds like a great idea! I think it would take a decent amount of work to get this working, but it's definitely doable. The healing tab would need to be adjusted so that it would: 

1. Detect out of combat status,  
2. Heal the player while resources are still available for healing. 

We'd need to keep the current behavior the same during battle where the player casts a mix set of spells (healing, battle, weaponskills), and then when out of combat, the player will idle and cast healing spells to heal up completely before the next fight. 

It may take some fine tuning and adjustment to get those things working together in just the right way.

**Comment by Mykezero** [comment]
Does that sound like it could do the trick? Hopefully I understood the problem this time ^^

**Comment by telostia** [comment]
Wow. I haven't been on github for awhile and came back and theres a new feature! That's exactly what I wanted. Thanks for your help! now to test out this feature :)

**Comment by telostia** [comment]
NO. the heal in my case here is the resting feature not the cure feature. like to rest up mps after battle
